data/mods/Aftershock/suit_operating_time.md

```diff
@@ -0,0 +1,106 @@
+ |        1 hours        | Doesn't grant any mental/knowledge related spells or enchantments        |
+ |        2 hours        | Has a non swappable integral battery.                                   |
+ |        2 hours        | Wont grant night/heat-vision or clairvoyance effects.                   |
+
+Once you have determined the operating time using the table above, simply divide the number 277777.78 by the indicated number of hours to obtain your new suit's `power_draw` json value.
+
+For player convenience, the power draw and battery capacity of all clothing pieces that complete a single matching outfit (for example: The magellan exosuit and its helmet) should be the same. 
+
+#### Example Calculation
+
+Consider the Magellan Exosuit, which has the following definitions:
+```
+  {
+    "id": "afs_magellan_suit",
+    "type": "TOOL_ARMOR",
+    "category": "armor",
+    "name": { "str": "Magellan exosuit" },
+    "description": "A high-quality, civilian grade EVA suit often employed by well-established frontier research and exploration associations.  Designed to support the exploration of challenging terrain, it offers respectable protection against common environmental hazards like extreme temperatures, inhospitable atmospheres, and  light radiation.  It leaves arms and hands relatively unencumbered to facilitate the manipulation of scientific instruments.\n\nAn integral battery allows the suit to operate for up to 36 hours, but complicates on the field recharging",
+    "weight": "7800 g",
+    "volume": "14 L",
+    "price": "4 kUSD",
+    "material": [ "nomex", "steel" ],
+    "symbol": "[",
+    "looks_like": "robofac_enviro_suit",
+    "color": "light_gray",
+    "ammo": "battery",
+    "charges_per_use": 1,
+    "use_action": {
+      "type": "transform",
+      "msg": "You activate your %s.",
+      "target": "afs_magellan_suit_on",
+      "active": true,
+      "need_charges": 1,
+      "need_charges_msg": "The %s's batteries are dead."
+    },
+    "armor_portion_data": [
+      { "covers": [ "torso" ], "coverage": 100, "encumbrance": 25 },
+      { "covers": [ "leg_l", "leg_r" ], "coverage": 100, "encumbrance": 25 },
+      { "covers": [ "arm_l", "arm_r" ], "coverage": 100, "encumbrance": 15 },
+      { "covers": [ "hand_l", "hand_r" ], "coverage": 100, "encumbrance": 10 },
+      { "covers": [ "foot_l", "foot_r" ], "coverage": 100, "encumbrance":  15 }
+    ],
+    "pocket_data": [
+      { "pocket_type": "MAGAZINE", "rigid": true, "ammo_restriction": { "battery": 1000 } }
+    ],
+    "warmth": 20,
+    "material_thickness": 2,
+    "valid_mods": [ "steel_padded" ],
+    "environmental_protection": 15,
+    "flags": [ "VARSIZE", "WATERPROOF", "GAS_PROOF", "POCKETS", "RAINPROOF", "STURDY", "RAD_RESIST",  "RECHARGE",  "OUTER", "NO_RELOAD", "NO_UNLOAD" ]
+  },
+
+```
+And whose active version provides no bonus except for cold protection.
```

Comment:
```suggestion
The active version provides no additional bonuses except for cold protection.
```
Stating the armor values (resulting from its thickness and materials) would be helpful.